components/service/item.vue

```diff
@@ -71,12 +44,14 @@
 </template>
 <script>
 import { PhoneIcon, LinkIcon } from '@heroicons/vue/solid'
+import RatingItem from '../rating/item.vue'
```

Comment:
all components in nuxt3 are auto-imported, see documentation

Reply:
yeah, i keep forgetting